scout/server/blueprints/cases/views.py

```diff
@@ -236,10 +235,16 @@ def generate_csv(header, lines):
                 headers.add(
                     "Content-Disposition",
                     "attachment",
-                    filename=clinvar_subm_id
-                    + "_"
-                    + str(datetime.datetime.now().strftime("%Y-%m-%d"))
-                    + ".csv",
+                    filename="".join(
```

Comment:
Another possibility could be to make an f-string. I would go for the option that is easiest to read though. Which one do you prefer?

Reply:
I can make it an f string if you want!

Reply:
I like it!

Reply:
Me too, thanks for the nice suggestion!